h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -540,6 +535,68 @@ public void exportModelsBinary(final String exportDir) throws IOException {
     }
   }
 
+  @SuppressWarnings("rawtypes")
+  private Set<Key> getDependentKeysFromParams() {
+    Field[] fields = Weaver.getWovenFields(getParams().getClass());
+    Set<Key> values = new HashSet<>();
+    for (Field f : fields) {
+      f.setAccessible(true);
+      Class<?> c = f.getType();
+      try {
+        Object value = f.get(getParams());
+        if (value instanceof Key) {
+          values.add((Key) value);
+        } else if (value != null && c.isArray() && c.getComponentType() == Key.class) {
+          Key[] arr = (Key[]) value;
+          for (Key k : arr) 
+            if (k != null) values.add(k);
+        }
+      } catch (IllegalAccessException e) {
+        throw new RuntimeException(e);
+      }
+    }
+    return values;
+  }
+  
+  private Job persistObj(
+      final Keyed o,
+      final String exportDir,
+      Map<String, String> referenceKeyTypeMap
+  ) {
+    if (o instanceof Frame) {
+      referenceKeyTypeMap.put(o._key.toString(), "frame");
+      return new FramePersist((Frame) o).saveTo(exportDir, true);
+    } else if (o != null) {
+      referenceKeyTypeMap.put(o._key.toString(), "iced");
```

Comment:
What would this be?

Reply:
pretrained_autoencoder in DeepLearning for example

Reply:
in this case, it would only write DeepLearningModel but not actually the weights and biases (that are in DKV), am I right?